### DISCUSSION TITLE
Release date for v1.0?

### DISCUSSION THREAD

**Discussion by matt-kg** [argoproj/argo-rollouts#1195] [Q&A] [answered]
Hi!
I can see that v1.0 is almost done (according to roadmap and the RC tag on the repo). Did a quick search and couldn't find any mention of when v1.0 might be released though, so apologies if that info is out there and I missed it.

Wondering about a ballpark date for the v1.0 release. Currently using v0.9.1 and am looking to upgrade before a project goes live but would prefer to update only once before then (I don't manage the clusters so there's some organizational overhead to this upgrade). If v1.0 is due in the next few weeks, I can hold out for it but it's going to be more like a month or two, will go with v0.10.2.

Thanks,
-Matt

**Comment by jessesuen** [comment, marked as answer]
v1.0 is slated for this week!

**Reply by matt-kg** [reply]
Thanks, that's perfect - I'll hold off until next week and then give it a go
